### PR TITLE
Board support for Adafruit Trinkey QT2040

### PR DESCRIPTION
**Pull request by dylanede**
Includes rainbow example. Tested on a board plugged into a Pi 4 Model B.

### CODE REVIEW

File: boards/adafruit-trinkey-qt2040/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[package]
+name = "adafruit-trinkey-qt2040"
+version = "0.1.0"
+authors = ["Dylan Ede <dylanede@googlemail.com>"]
```
Comment (thejpster) [inline boards/adafruit-trinkey-qt2040/Cargo.toml:4]: Would you mind setting this to "The rp-rs Developers"? I'm trying to get them all changed to be the same for consistency.

[See https://github.com/rp-rs/rp-hal/pull/245](https://github.com/rp-rs/rp-hal/pull/245#issuecomment-1000231052)

File: boards/adafruit-trinkey-qt2040/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# [adafruit-trinkey-qt2040] - Board Support for the [Adafruit Trinkey QT2040]
+
+You should include this crate if you are writing code that you want to run on
+an [Adafruit Trinkey QT2040] - a 3x4 keyboard and OLED combo board from Adafruit.
+
+This crate includes the [rp2040-hal], but also configures each pin of the
+RP2040 chip according to how it is connected up on the Feather.
```
Comment (thejpster) [inline boards/adafruit-trinkey-qt2040/README.md:7]: err: Feather. Should be "Trinkey"?

**Review by thejpster — CHANGES_REQUESTED** [review]
Thank you so much for this PR! I just had one nit and one question, otherwise good to go! :)

**Comment by thejpster** [comment]
Sorry - one more thing. Please could you add it to the top-level README along with the other boards?

File: boards/adafruit-trinkey-qt2040/Cargo.toml (hunk 1 above)
Comment (dylanede) [reply boards/adafruit-trinkey-qt2040/Cargo.toml]: Sure

File: boards/adafruit-trinkey-qt2040/README.md (hunk 2 above)
Comment (dylanede) [reply boards/adafruit-trinkey-qt2040/README.md]: Oops, yes.

**Comment by dylanede** [comment]
Requested changes made.

**Review by thejpster — APPROVED** [review]
Thank you! Looks great.

**Pull request merged**